### PR TITLE
Minibatch Preprocessor: Update online doc

### PR DESCRIPTION
**Pull request by njayaram2**
The online doc is outdated. This commit adds two new parameters that
have been introduced since the last time the doc was edited.

### CODE REVIEW

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/madlib-pr-build/705/

File: src/ports/postgres/modules/utilities/minibatch_preprocessing.py_in [hunk 1]
```diff
@@ -508,8 +514,13 @@ class MiniBatchDocumentation:
             dependent_varname,     -- TEXT. Name of the dependent variable column
             independent_varname,   -- TEXT. Name of the independent variable
                                       column
-            buffer_size            -- INTEGER. Number of source input rows to
-                                      pack into batch
+            grouping_col           -- TEXT. Default NULL. An expression list used
+                                      to group the input dataset into discrete groups
+            buffer_size            -- INTEGER. Default computed automatically.
+                                      Number of source input rows to pack into batch
```
Comment (kaknikhil) [inline src/ports/postgres/modules/utilities/minibatch_preprocessing.py_in]: /s/batch/buffer

File: src/ports/postgres/modules/utilities/minibatch_preprocessing.py_in [hunk 2]
```diff
@@ -487,10 +487,16 @@ class MiniBatchDocumentation:
...
+        The mini-batch preprocessor is a utility that prepares input data for
+        use by models that support mini-batch as an optimization option. (This
+        is currently only the case for Neural Networks.) It is effectively a
+        packing operation that builds arrays of dependent and independent
+        variables from the source data table.
 
-        #TODO add more here
+        The advantage of using mini-batching is that it can perform better than
+        stochastic gradient descent (default MADlib optimizer) because it uses
+        more than one training example at a time, typically resulting faster
```
Comment (kaknikhil) [inline src/ports/postgres/modules/utilities/minibatch_preprocessing.py_in]: missing the word in `resulting in faster .....`

File: src/ports/postgres/modules/utilities/minibatch_preprocessing.py_in [hunk 3]
```diff
@@ -487,10 +487,16 @@ class MiniBatchDocumentation:
         ----------------------------------------------------------------
                             SUMMARY
         ----------------------------------------------------------------
-        MiniBatch Preprocessor is a utility function to pre process the input
-        data for use with models that support mini-batching as an optimization
+        The mini-batch preprocessor is a utility that prepares input data for
+        use by models that support mini-batch as an optimization option. (This
+        is currently only the case for Neural Networks.) It is effectively a
```
Comment (kaknikhil) [inline src/ports/postgres/modules/utilities/minibatch_preprocessing.py_in]: /s/Neural Networks/Neural Network

File: src/ports/postgres/modules/utilities/minibatch_preprocessing.py_in [hunk 4]
```diff
@@ -487,10 +487,16 @@ class MiniBatchDocumentation:
...
+        packing operation that builds arrays of dependent and independent
```
Comment (kaknikhil) [inline src/ports/postgres/modules/utilities/minibatch_preprocessing.py_in]: should we instead say `build matrix of independent variable(s) and arrays of dependent variable` ?

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/madlib-pr-build/710/

**Pull request closed without merging**